src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -817,6 +817,17 @@ class CreateVmModal extends React.Component {
         case 'storageVolume':
             this.setState({ [key]: value });
             break;
+        case 'memorySize': {
+            // virt-install doesn't allow memory, supplied in MiB, to be a float number
+            // check if number is a integer, and if not, round up (ceil)
+            let valueMiB = convertToUnit(value, this.state.memorySizeUnit, units.MiB);
+            if (valueMiB % 1 !== 0)
+                valueMiB = valueMiB.ceil();
```

Comment:
Comparing floating point numbers to exact integers is unreliable in general, even if it should work for the special case of 0. Let's just drop the `if()`, and always call ceil(), it doesn't hurt. Then you can also make `valueMiB` `const`.

Also, I've never seen this being used as a method of a number object, and this doesn't work for me in node:
```
 x = 3
3
> x.ceil()
Uncaught TypeError: x.ceil is not a function
> 
```

If that works in some browsers, let's not rely on that. Please use standard [Math.ceil() function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/ceil)